sys/hashes/cmac.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_hashes
+ * @{
+ *
+ * @file
+ * @brief       AES_CMAC implementation
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ *
+ * @}
+ */
+
+
+#include <stdio.h>
+#include "hashes/cmac.h"
+#include <inttypes.h>
+#include "crypto/ciphers.h"
+#include <string.h>
+
+#define XOR128(x, y) do { \
```

Comment:
Could you try to make this a static inline function and see if it would increase codesize?